localstack-core/localstack/http/resources/swagger/templates/index.html

```diff
@@ -0,0 +1,19 @@
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Swagger UI</title>
+    <link rel="stylesheet" type="text/css" href="/static/swagger-ui.css" />
+    <link rel="stylesheet" type="text/css" href="/static/index.css" />
+    <link rel="icon" type="image/png" href="/static/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="/static/favicon-16x16.png" sizes="16x16" />
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+    <script src="/static/swagger-ui-bundle.js" charset="UTF-8"> </script>
+    <script src="/static/swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
+    <script src="/_localstack/swagger-initializer.js" charset="UTF-8"> </script>
+  </body>
+</html>
```

Comment:
Consider using [swagger-ui-dist](https://www.npmjs.com/package/swagger-ui-dist) which will eliminate the need to dump the entire swagger UI code in our codebase

https://github.com/swagger-api/swagger-ui/blob/HEAD/docs/usage/installation.md#unpkg

Reply:
Thanks 🙌 I overlooked this!